styles.css

```diff
@@ -532,3 +531,113 @@ textarea {
   justify-content: end;
   padding: 25px 0;
 }
+
+/* Project Details */
+
+.projectDetailContainer {
+  background-color: rgba(0, 0, 0, 0.4);
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+}
+
+.projectDetail {
+  background-color: #fff;
+  width: 90%;
+  height: 90vh;
+  position: fixed;
+  transform: translate(-50%, -50%);
+  z-index: 2;
+  border-radius: 2rem;
+  top: 50%;
+  left: 50%;
+  padding: 1.7rem;
+  overflow-y: scroll;
+}
+
+.projectDetail h2 {
+  color: #172b4d;
+  font-size: 2.4rem;
+}
+
+.projectDetails__h1Div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 1rem;
+}
+
+.projectDetails__buttons {
+  display: flex;
+  justify-content: center;
+  gap: 1.7rem;
+}
+
+.projectDetails__button {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  gap: 1rem;
+  color: #396df2;
+  border: 1px solid #396df2;
+  background-color: white;
+  border-radius: 8px;
+  padding: 1.6rem 2.6rem;
+  font-size: 1.7rem;
```

Comment:
- [ ] - [ ] Nice work with the dynamic popups :clap: , however please notice that we have to stick to the figma template as much as possible, I think it will be better if you use the correct font size and padding for your popup buttons in the mobile popup, currently they are like :point_down: 
![image](https://user-images.githubusercontent.com/97521557/216128081-2e804316-8aa9-4a2c-83be-aa7feb3f57a5.png)

by applying the correct typography which is   
```
padding: 12px 14px;
font-size: 15px;
```
we can make it like this :point_down: 
![image](https://user-images.githubusercontent.com/97521557/216128651-35b45c81-76a0-4152-a7af-98b2fad88a8b.png)

which is much better and closer to figma :+1: